Makefile

```diff
@@ -24,9 +24,9 @@ $(EMBEDDED_STATIC): static/kubeapps-objs.yaml
 
 static/kubeapps-objs.yaml:
 	KUBEAPPS_VERSION=$${VERSION:-latest} ;\
-	pushd manifests/vendor/kubeless ;\
+	cd manifests/vendor/kubeless ;\
 	KUBELESS_VERSION=$$(git describe --tags --abbrev=0) ;\
-	popd ;\
+	cd ../../../ ;\
```

Comment:
`cd -` ?

Reply:
thanks, let's see if that works in travis